eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java

```diff
@@ -92,7 +96,8 @@ public boolean login() throws LoginException {
 
             return true;
         } catch (IOException | UnsupportedCallbackException | LoginException e) {
-            throw new LoginException(e.getMessage());
+            logger.log(Level.SEVERE, "login() method failed", e);
```

Comment:
\[Checkstyle\] WARNING: 'catch' child have incorrect indentation level 12, expected level should be 6\.